### PR TITLE
feat(ci): add proto and playwright image builder

### PR DESCRIPTION
**Pull request by c3ppc3pp**

### CODE REVIEW

File: web/crux-ui/e2e/utils/node-helper.ts [hunk 1]
```diff
@@ -65,6 +65,13 @@ export const deployWithDagent = async (
   await navigation
 
   if (!ignoreResult) {
+    await page.waitForTimeout(1000)
+    await page.screenshot({ path: screenshotPath('should-successful-after-1k'), fullPage: true })
```
Comment (polaroi8d) [inline web/crux-ui/e2e/utils/node-helper.ts:69]: Instead of `...-1k` we have to use `...-1s`, so the reader easily can identify the seconds for it, IMHO

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: images/proto/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
[the remark this answers is not in the document]
Comment (c3ppc3pp) [reply images/proto/package.json:1]: You're damn right. Will fix.

File: web/crux-ui/e2e/utils/node-helper.ts (hunk 1 above)
Comment (c3ppc3pp) [reply web/crux-ui/e2e/utils/node-helper.ts:69]: These are temporary stuff. Seems we can finally strike off the last flaky test. We're working on it with @m8vago. After I clean up, please remind me to remove these lines if I forget to.

File: images/proto/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (c3ppc3pp) [reply images/proto/package.json]: Fixed.

File: web/crux-ui/e2e/utils/node-helper.ts (hunk 1 above)
Comment (polaroi8d) [reply web/crux-ui/e2e/utils/node-helper.ts]: Resolved.

**Review by polaroi8d — APPROVED** [review]

**Pull request merged**